features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/org.wso2.carbon.identity.application.authentication.framework.server.feature.default.json

```diff
@@ -67,6 +67,7 @@
   "authentication.authenticator.basic.enable": true,
   "authentication.authenticator.basic.parameters.AuthMechanism": "basic",
   "authentication.authenticator.basic.parameters.Tags": "Username-Password",
+  "authentication.authenticator.basic.parameters.redirectToRetryPageOnAccountLock": "false",
```

Comment:
Default behavior is getting changed now right? In that case we should mention it in migration doc

Reply:
Default behavior will not be changed until redirectToRetryPageOnAccountLock is enabled.